lib/rules/jsx-no-target-blank.js

```diff
@@ -33,8 +33,21 @@ module.exports = {
           var relFound = false;
           var attrs = node.parent.attributes;
           for (var idx in attrs) {
-            if (attrs[idx].name && attrs[idx].name.name === 'rel') {
-              var tags = attrs[idx].value.type === 'Literal' && attrs[idx].value.value.toLowerCase().split(' ');
+            if (!Object.prototype.hasOwnProperty.call(attrs, idx)) {
```

Comment:
in `forEach()`, only there's only`return` available to control loops, but this loop needs `return`, `continue` and `breaks`, so `forEach()` is not suitable.